docs/marketplace/dev/front-end/table-design/table-configuration.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Table Configuration
+description: This articles provides details about the table configuration.
+template: concept-topic-template
+---
+
+This articles provides details about the table configuration.
+
+## Configuration
+
+Table Configuration is basically a data structure that allows to customize behavior of the table for specific use-case.
+
+Table config has 2 required sections:
+
+- [Column](#columns-configuration) definition that describes what columns user will see and what data to expect
+- [Datasource](/docs/marketplace/dev/front-end/ui-components-library/datasources/index.html) type that describes how the data should be provided to the table.
+
+The rest of the sections are reserved for features (for example Pagination Feature defines pagination section and it’s own properties to configure it).
+
+```html
+<spy-table
+  config="{
+    columns?: '### Section Columns Configuration ###',
+    columnsUrl?: '### Section Columns Configuration ###',
+    dataSource: '### See DataSource Doc ###',,
+    rowActions: '### See Table Row Actions Feature Doc ###',
+    pagination: '### See Table Pagination Feature Doc ###',
+    search: '### See Table Search Feature Doc ###',
+    title:: '### See Table Title Feature Doc ###',
+    columnConfigurator: '### See Table Settings Feature Doc ###',
+    filters: '### See Table Filters Feature Doc ###',
+    itemSelection: '### See Table Selectable Feature Doc ###',
+    syncStateUrl: '### See Table Sync State Feature Doc ###',
+    editable: '### See Table Editable Feature Doc ###',
+  }"
+>
+</spy-table>
+```
+
+### Columns Configuration
+
+Columns might be configured with two props. There is possible to use `url` to get columns list dynamic, in this case you should assign `columnsUrl` property in the `table` config.
+Also exists another way to define columns data - static array of columns objects which should be assigned to the `columns` property.
+
+```ts
+[
+  // chip
+  {
+    "id":"stock",
```

Comment:
I think it should be mentioned that it's for example because there are described not all types.